python/foglamp/services/core/api/configuration.py

```diff
@@ -201,6 +201,8 @@
 
     try:
         value = data['value']
+        if not isinstance(value, str):
```

Comment:
or may be  just `value = str(value)`

Reply:
I would tend to agree, turn it into a string if it is not. so { "value" : 10 } and { "value" : "10" } with work. The most important thing is that we raise an error if the value is not stored.